github_strategy.md

```diff
@@ -0,0 +1,166 @@
+
+When people think about github, they usually think about open source software packages. The lab often produces individual pieces of software that we and the community use as tools. We follow standard procedures when writing software packages including:
+
+- Language-specific file structure and organization
+
+- Portable to packaging engines (e.g. pip, conda)
+
+- Maximal test coverage with continuous integration (using GitHub actions)
+
+- Consistent functional API
+
+- Well-documented README introducing the software
+
+- Full auto-doc generating a complete read-the-docs page (e.g. sphinx)
+
+- Gallery outlining example use cases
+
+Examples: [pycytominer](https://github.com/cytomining/pycytominer), [CytoSnake](https://github.com/WayScience/CytoSnake)
+
+### 2. Analysis repository
+
+As biomedical data scientists, analysis repositories are the most common type of repository for our group. When most people think about a github repository, they are likely not thinking about storing a reproducible analysis. Nevertheless, this approach is gaining in popularity. 
+
+An analysis repository contains a collection of analyses linked together by the name of the project, which should be descriptive of the specific set of interrelated hypotheses.
+
+Analysis repositories contain numbered folders, or modules, that have a sequential relationship. For example, the first step in most analysis repositories is data access. Therefore, the first module is often called \`0.download-data\`, and it contains instructions on how to access the data you use in your analysis. The subsequent modules all use these data in various ways; often exploring the data or testing various hypotheses. We also include code for generating all publication quality figures as the last module. 
+
+We use software libraries in these analyses, and we save specific versions in conda environment files or docker containers. We also must specify the license, present a clear README with reproducibility/usage instructions, a brief summary of the results, and citation instructions.
+
+Analysis repositories must be included in our publications, and we consider them to be the ground truth methods section. The analyses must be fully reproducible (we aim for [Gold Reproducibility](https://www.nature.com/articles/s41592-021-01256-7#Sec4)) and open source. For each submission and subsequent resubmission, we generate specific “github releases”. We generate direct object identifiers (DOI) for these releases using Zenodo ([instructions](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content)). We do not restrict ourselves to the one-repository per-paper strategy, instead, we opt for the one-release per-submission strategy (which can span multiple publications). 
+
+Examples: [phenotypic\_profiling\_model](https://github.com/WayScience/phenotypic_profiling_model), [gene\_dependency\_representations](https://github.com/WayScience/gene_dependency_representations)
```

Comment:
Recommend updating examples to include recent publication repositories like NF1 and pyroptosis which are relevant to this section.

Reply:
Could you point me to the correct repos to include here? I'm seeing multiple repos for each of these topics

Reply:
I think the repos currently mentioned here are both somewhat outdated for Way Lab standards. Maybe it would make more sense to just include one example of a bioinformatics pipeline/analysis and one example of a data repo from the same publication?

If you agree, which set of repos would you recommend including here @jenna-tomkinson?

Reply:
I will provide NF1 as the example since it is the most recent 😄 

- Data/image profiling repo: https://github.com/WayScience/nf1_schwann_cell_painting_data

- ML/bioinformatics pipeline repo: https://github.com/WayScience/nf1_schwann_cell_morphology_signature

Reply:
Done in https://github.com/WayScience/playbooks/pull/1/commits/cb7729e0b961f300ef99b1d17bcc0efb0da24475, thanks!